.github/workflows/py-bench.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         run: |
           {
             echo 'OUTPUT<<EOF'
-            make -s benchmark
+            make -s benchmark-fast
```

Comment:
Maybe make configurable & run full one on cron?

Reply:
this only runs on pull requests, not cron

let's do cron separate if we want it. would want that one hooked up to some sort of slack notify probably